frontend/src/helpers/validators/StaffFormValidations.ts

```diff
@@ -229,6 +232,55 @@ fieldValidations["location.addresses.*.notes"] = [
   isMaxSizeMsg("notes", 4000),
   isAsciiLineBreak("notes"),
 ];
+fieldValidations["location.addresses.*.locationName"] = [
```

Comment:
These validations are identical to others within the application. To adhere to the DRY (Don't Repeat Yourself) principle, they should be consolidated into a common file. This will enhance maintainability and consistency across the codebase.

Reply:
Noted, will be done in a separate branch